### PR TITLE
Fix shellcheck failures in intemp.sh

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref ##72956

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by fejta — APPROVED** [review]
/lgtm
/approve

**Comment by xichengliudui** [comment]
here.

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Review by fejta — COMMENTED** [review]
/approve

Please rebase

**Comment by xichengliudui** [comment]
Could you please tell me what I should do according to git's workflow? (these days, I have to close a lot of pr because they need rebase and open a new pr).

**Comment by fejta** [comment]
`git remote add upstream https://github.com/kubernetes/kubernetes.git && git fetch upstream && git rebase upstream/master`

https://git-scm.com/docs/git-rebase

**Comment by xichengliudui** [comment]
@fejta Thank you very, very much. I think it should be ok this time.

File: third_party/intemp/intemp.sh
Comment (BenTheElder) [reply third_party/intemp/intemp.sh]: /hold
@mrbobbytables what happened to updating the verify script to ignore third_party except for the forked files?
this is our local, unmodified copy of an external script, modified scripts are in third_party/forked.

Comment (mrbobbytables) [reply third_party/intemp/intemp.sh]: Needs lgtm after rebase the other day - https://github.com/kubernetes/kubernetes/pull/75795

Comment (BenTheElder) [reply third_party/intemp/intemp.sh]: thanks, LGTM

**Comment by BenTheElder** [comment]
Thank you for the PR, but we should do https://github.com/kubernetes/kubernetes/pull/75795 instead.

**Comment by BenTheElder** [comment]
this was superseded by https://github.com/kubernetes/kubernetes/pull/75795
thanks again.
/close

**Pull request closed without merging**
